openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -321,11 +321,12 @@ def test_get_account_empty_string(self):
         legacy_profile.country = ""
         legacy_profile.level_of_education = ""
         legacy_profile.gender = ""
+        legacy_profile.bio = ""
```

Comment:
Not a change in behavior. All 4 attributes are converted to none by the legacy serializer. We were just testing 3/4 of them.

```python
    def transform_bio(self, user_profile, value):
        """ Converts empty string to None, to indicate not set. Replaced by to_representation in version 3. """
        return AccountLegacyProfileSerializer.convert_empty_to_None(value)
```